app_dart/lib/src/request_handlers/presubmit_luci_subscription.dart

```diff
@@ -175,7 +177,8 @@ class PresubmitLuciSubscription extends SubscriptionHandler {
     final target = targets.singleWhere((element) => element.value.name == builderName);
     final Map<String, Object> properties = target.getProperties();
     if (!properties.containsKey('presubmit_max_attempts')) {
-      return 1;
+      // Give any test in the merge queue another try... its expensive otherwise.
+      return tags.any((pair) => pair.key == LuciBuildService.kMergeQueueKey) ? 3 : 1;
```

Comment:
Overheard your discussion with Yegor, and given that the test is `<` above, I agree this should probably be 4 instead of 3 in order to get 3 attempts.

Reply:
Will do